elements/lisk-api-client/src/transaction.ts

```diff
@@ -234,7 +246,13 @@ export class Transaction {
 
 	public computeMinFee(transaction: Record<string, unknown>): bigint {
 		const assetSchema = getTransactionAssetSchema(transaction, this._schema);
-		return computeMinFee(assetSchema, transaction);
+		const options: Options = {
+			minFeePerByte: this._nodeInfo.genesisConfig.minFeePerByte,
+			baseFees: this._nodeInfo.genesisConfig.baseFees,
+			numberOfSignatures: (transaction.asset as { numberOfSignatures: number }).numberOfSignatures || 1,
```

Comment:
shouldnt this be `transaction?.signatures.length > 0 ? transaction?.signatures.length : 1`
or accept as optional params as in the `computeMinFee`